pydoctor/test/test_epydoc2stan.py

```diff
@@ -171,8 +177,8 @@ def f(a, b):
         @rtype: C{bool}
         """
     ''')
-    annotation_fmt = docstring2html(annotation_mod.contents['f'])
-    classic_fmt = docstring2html(classic_mod.contents['f'])
+    annotation_fmt = docstring2html(annotation_mod.contents['f']).replace('<wbr></wbr>', '').replace('<wbr>\n</wbr>', '')
```

Comment:
Like for `type2html()`, the `<wbr>` removal could be added to `docstring2html()` instead of having to repeat it in the test cases.